_data/sidebars/main_sidebar.yml

```diff
@@ -51,16 +51,16 @@ entries:
       url: analyze-source-code.html
     - title: Accessing Source Code Premium Insights
       output: web
-      url: /source_code_insights.html
+      url: /source-code-insights.html
```

Comment:
Why do some of these URLs have a forward slash and others don't? Are they a nested route?

Reply:
no good reason, both are valid. we have been inconsistent when adding them.